examples/hotrod/services/driver/redis.go

```diff
@@ -53,7 +53,7 @@ func (r *Redis) FindDriverIDs(ctx context.Context, location string) []string {
 		span.SetTag("param.location", location)
 		ext.SpanKindRPCClient.Set(span)
 		defer span.Finish()
-		ctx = opentracing.ContextWithSpan(ctx, span)
+		_ = opentracing.ContextWithSpan(ctx, span)
```

Comment:
This is wrong. The context is used later.

Reply:
Actually it looks like this is correct. It is not used again in the existing method and the `ctx` argument is copied by value (not a pointer). Unless context is an interface or something like that, cannot see how this works.

Reply:
I guess it is an interface: https://golang.org/pkg/context/#Context.

Reply:
Hard for me to verify on the phone, but isn't the context later used for logging?